scripts/lp-create-bug

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+"""Mark the provided launchpad bugs as fix-released.
+
+Update the bug with a comment.
```

Comment:
This confused me a bit, then I realized it comes straight from `lp-bugs-released`. It should probably be adapted to what this script is actually doing.

Reply:
Fixed. also sourced the module __doc__ in the argparser created